ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs

```diff
@@ -8,11 +8,6 @@ static class ViewManagersPropertyCache
     {
         private static readonly IReadOnlyDictionary<string, IPropertySetter> s_shadowEmpty = new Dictionary<string, IPropertySetter>();
 
-        private static readonly object[] s_viewArgsBuffer = new object[2];
```

Comment:
While you're at it, can you move this file back under UIManager folder where it belongs?